lib/active_fedora/file.rb

```diff
@@ -147,12 +167,13 @@ def to_solr(solr_doc = {}, _opts = {})
     end
 
     def content=(string_or_io)
+      raise "This file has external content.  First call external_url=nil if you want to use content=." if external_content?
       content_will_change! unless @content == string_or_io
       @content = string_or_io
     end
 
     def content
-      local_or_remote_content(true)
+      external_content? ? external_content : local_or_remote_content(true)
```

Comment:
Do we need to explicitly retrieve the external content URL from the description, and then retrieve it?  Can't we just follow Fedora's redirect?